charts/adaptive/values.yaml

```diff
@@ -156,6 +156,27 @@ controlPlane:
 
   extraEnvVars: {}
 
+# Sandboxing service
+sandkasten:
+  servicePort: 3005
+  podAnnotations:
+    ingest-adaptive-logs: "true"
+  podLabels: {}
+  nodeSelector: {}
+  tolerations: {}
+  extraEnvVars: {}
+  replicaCount: 1
+  image:
+    repository: adaptive-repository   # Add the Adaptive Repository you have been granted access to
+    tag: latest   # Add the sandkasten image tag
+    pullPolicy: Always
```

Comment:
I think its going to build an image looking like
`<aws_account_id>.dkr.ecr.<region>.amazonaws.com/adaptive-repository:latest`.
With the current helm values. `.Values.containerRegistry .Values.sandkasten.image.repository .Values.sandkasten.image.tag`

I think default repository should probably be `something like `adaptive/sankasten`.

I think the tag policy with this image should be aligned with the other images instead of pulling a `latest` image which could create breaking changes for our users.